Core/Utilities/Pagination/PaginationParameter.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace WebNetSample.Core.Pagination;
+
+public class PaginationParameter : PaginationParameters
```

Comment:
Not really sure about purpose of `PaginationParameter` since it does not extend any functionality of base class. Is it necessary to have? I think we can skip it (at least for now) and use just `PaginationParameters`